src/containers/App/index.test.tsx

```diff
@@ -60,13 +56,13 @@ describe('<App />', () => {
   it('should scroll to top on history change', () => {
     render(withAppContext(<App />))
 
-    expect(spyScrollTo).not.toHaveBeenCalled()
+    expect(spyScrollTo).toHaveBeenCalledWith(0, 0)
```

Comment:
why is the scrollTo called on initialize?

Reply:
because we don't have the redirect component anymore, we are now redirecting by first hitting '/', then as an element we serve navigate which navigates to the new route. Perf should be better though in RR6.